docs/source/release_notes/release-notes.rst

```diff
@@ -49,6 +49,9 @@ develop
          - Fixed an issue where some locks were not being tracked for continuous refreshing due to one of the lock methods not being
            overridden by the ``LockRefreshingLockService``.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1134>`__)
+    *    - |improved|
+         - all KVSs now as a guarantee throw a RuntimeException on attempts to truncate a non-existing table
```

Comment:
This will be breaking eg: for Oracle truncate a table that did not exist(It executes ignoring ORACLE_NOT_EXISTING_ERROR) and the application will keep running after that, but now it will halt. Should we stop ignoring the error there too?